doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -883,6 +883,10 @@ It returns a list of versions with their metadata, and file list:
     ]
   }
 
+The optional ``includeFiles`` parameter specifies whether the files should be listed in the output. It defaults to ``false``. (Note that for a dataset with a large number of versions and/or files having the files included can dramatically increase the volume of the output). A separate ``/files`` API can be used for listing the files, or a subset thereof in a given version. 
```

Comment:
This sounds like a backward incompatible change. Can you please add a release note snippet about this?

(I'm a little nervous about what integrations will break.)

Reply:
Good point, thank you. It IS actually backward-compatible, as submitted in the PR. (i.e., `includeFiles` defaults to `true` as implemented). But the guide fragment needs to be changed to reflect that.

Reply:
I have also added a release note.

Reply:
Thanks, I'm glad we're preserving backward compatibility.